blobstore/blobhandler.go

```diff
@@ -323,6 +323,14 @@ func (bh *BlobHandler) HandleCheckS3UserPermission(c echo.Context) error {
 		log.Info("Checked user permissions successfully")
 		return c.JSON(http.StatusOK, true)
 	}
+	initAuth := os.Getenv("INIT_AUTH")
+
+	if initAuth == "0" {
```

Comment:
I am not sure how this would ever be true? If initAuth == "0" then line 324 would have already returned

Reply:
This is the new init auth that will turn on and off auth in general, not the auth level which indicates whether we want FGAC or not. This is because when auth is turned off we can't get the user email from the claims which would break this endpoint. 
I've just added checks to two other functions that are used in endpoints that require the claims.